libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -954,7 +952,8 @@ void ContractCompiler::handleCatch(vector<ASTPointer<TryCatchClause>> const& _ca
 				revert(0, returndatasize())
```

Comment:
One idea here could be to check whether `returndatasize()` is 0, and if yes, add string `Uncaught exception` or something like that. Not sure if too pedantic though.